src/components/DateRangePickerInput.jsx

```diff
@@ -153,6 +158,7 @@ function DateRangePickerInput({
   customArrowIcon,
   customCloseIcon,
   isFocused,
+  datePickerIsAfterEndInput,
```

Comment:
what's the propType and defaultProp for this?

Reply:
Also can we not calculate this based on the isStartDateFocused/isEndDateFocused props? Do we need this new one?

Reply:
@majapw Maybe. Does `isEndDateFocused` take into account whether the date input is focused, or just the day picker widget?

Reply:
@ljharb It's not added, but this isn't ready for review yet - either. I was just asked by @majapw to put the PR up anyway :)